package.json

```diff
@@ -9,9 +9,10 @@
     "crypto"
   ],
   "author": "WalletConnect <walletconnect.org>",
-  "homepage": "https://walletconnect.org",
+  "homepage": "http://localhost:3000/testwallet",
   "license": "LGPL-3.0",
   "scripts": {
+    "serve-start": "serve -s build",
```

Comment:
what's this doing?

Reply:
Over in our nori-cms serve.js, I use the following to start the testwallet:

Reply:
Right, I guess the question should have been what is this new dependency bringing to the table that the existing create react app scripts don't.

My thinking is if we avoid adding dependencies to this fork then we avoid merge messes with yarn.lock which will make it significantly easier to keep this fork up to date.

Reply:
What's the right way to manage it?